### ISSUE TITLE
The gameplay is very difficult very fast.

### ISSUE THREAD

**Issue by ehgoodenough** [mocsarcade/starjunk#105]
I like the pacing of the game, but not when you first start. Any ideas on making the game easier to ease into?

I still advocate for one-hit-one-life, but if it would make the game more forgiving to take multiple hits or have multiple lives, I'm willing to entertain any suggestions.

**Comment by codaventure** [comment]
The game is also actually incredibly easy when played cooperatively if one player destroys all the Trashbots while the other collects Junk. So, ideally we need to make the game easier in individual play at the beginning while ensuring it's not any easier in cooperative play.

So, one thing we could do is multiple hit death for players, or we could make enemy movement slower and more predictable? Also maybe making enemy projectiles easier to avoid initially, since projectiles are most difficult as difficulty progresses. Another approach could be to make enemies have intelligent movement, but less numerous initially. Furthermore, their intelligent movement could go as far as even trying to protect Junk that was dropped, maybe?

Additional thoughts?

**Comment by ehgoodenough** [comment]
I agree, the number of enemies is something that could use some work. Maybe replace the spawning algorithm we have at the moment with an array of scripted waves. Each wave is JSON that the game pulls from for what enemies to spawn.

I think we keep the spawning algorithm, but only start using it after Level 10 (or whatever), after guiding the players through the scripted waves.
